### PR TITLE
Nursery back to front

### PR DESCRIPTION
**Pull request by EmilieLgt**

### CODE REVIEW

File: client/src/components/NurseriesPlateform/NurseriesAllLille.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-unused-vars */
```
Comment (BENOITMEZAGUER) [inline client/src/components/NurseriesPlateform/NurseriesAllLille.jsx]: Can you please remove this commentary?

File: client/src/components/NurseriesPlateform/NurseriesAllRennes.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-unused-vars */
```
Comment (BENOITMEZAGUER) [inline client/src/components/NurseriesPlateform/NurseriesAllRennes.jsx]: Same thing as the previous files

File: server/app/controllers/nuseryActions.js [hunk 3]
```diff
@@ -3,10 +3,17 @@ const tables = require("../../database/tables");
 // The B of BREAD - Browse (Read All) operation
 const browse = async (req, res, next) => {
   try {
-    // Fetch all nurseries from the database
+    // Fetch all players from the database
```
Comment (BENOITMEZAGUER) [inline server/app/controllers/nuseryActions.js]: I think it's another project here

File: server/database/fixtures/NurserySeeder.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ class NurserySeeder extends AbstractSeeder {
         price: 10,
         nursery_phone: "03 20 85 96 41",
         nursery_mail: "contact@lillomomes.fr",
-        image1: "../../public/assets/images/nurseriesImages/0101.jpg",
+        image1: "./assets/images/nurseriesImages/0101.jpg",
```
Comment (BENOITMEZAGUER) [inline server/database/fixtures/NurserySeeder.js]: Relocate the file correctly

**Review by BENOITMEZAGUER — CHANGES_REQUESTED** [review]
Some changes needed

**Review by BENOITMEZAGUER — APPROVED** [review]
Neat

**Review by fantinerudent — APPROVED** [review]

**Pull request merged**